benchmark/sirun/plugin-bluebird/index.js

```diff
@@ -4,6 +4,7 @@ if (Number(process.env.USE_TRACER)) {
   require('../../..').init()
 }
 
+// eslint-disable-next-line n/no-missing-require
```

Comment:
Most occurrences of this are in benchmarks and tests. Should that rule maybe not apply to those?

Reply:
Good point. I've disabled them 👍